src/Config/packages.aget

```diff
@@ -9,13 +9,13 @@
       "DynamoVisualProgramming.WpfUILibrary": "3.0.0-beta6764",
       "DynamoVisualProgramming.ZeroTouchLibrary": "3.0.0-beta6764",
       "DynamoVisualProgramming.DynamoSamples": "3.0.0-beta6764",
-      "Greg": "3.0.0.2955",
-      "GregRevitAuth": "2.0.8696.26508",
+      "Greg": "3.0.0.3136",
+      "GregRevitAuth": "2.0.8717.29722",
```

Comment:
I'm guessing this doesn't exist yet. How do you know the version number before it's published?